src/common/event.h

```diff
@@ -481,7 +486,7 @@ enum EventType
     EVENT_OBJECT_BNUCLEAR   = 1060,
     EVENT_OBJECT_BPARA      = 1061,
     EVENT_OBJECT_BINFO      = 1062,
-    EVENT_OBJECT_BDESTROYER = 1063,
+    EVENT_OBJECT_BSAFE      = 1063,
```

Comment:
A little shame that we can't build Destroyer now but it's an issue to fix after UI refactoring.